fido-device-onboard.spec

```diff
@@ -0,0 +1,122 @@
+%define dracutlibdir %{_prefix}/lib/dracut
+%bcond_without check
+%global __cargo_skip_build 0
+%global __cargo_is_lib() false
+%global gitversion 43f7674dad4abc470cdab505fbf49f571539cea8
+
+Name:           fido-device-onboard
+Version:        0.1.0
+Release:        1%{?dist}
+Summary:        An implementation of the FIDO Device Onboard Specification written in rust
+
+License:        BSD 3
+URL:            https://github.com/fedora-iot/fido-device-onboard-rs/
+Source:         %{url}/archive/%{gitversion}/%{name}-rs-%{gitversion}.tar.gz
+Source1:        %{name}-rs-%{gitversion}-vendor.tar.gz
+
+ExclusiveArch:  %{rust_arches}
+# RHBZ 1869980
+ExcludeArch:    s390x i686 %{power64}
+
+BuildRequires: rust-toolset
+BuildRequires: systemd
+
+%description
+%{summary}.
+
+%prep
+%autosetup -n %{name}-rs-%{gitversion} -p1
+%cargo_prep -V 1
+
+%build
+%cargo_build
+
+%install
+install -D -m 0755 -t %{buildroot}%{_libexecdir}/fdo target/release/fdo-client-linuxapp
+install -D -m 0755 -t %{buildroot}%{_libexecdir}/fdo target/release/fdo-manufacturing-client
```

Comment:
Isn't libexecdir only for binaries that are being executed by other binaries?
These are either executed by hand, or by a systemd unit, and I thought that would have to go into /usr/bin?

Reply:
not fully sure, that's something I've always done for systemd unit cause those binaries aren't called by users (and systemd isn't a user right?)